account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940

```diff
@@ -0,0 +1,17 @@
+:20:6450374100
```

Comment:
Ah, OK, I made the tests independently from this: https://github.com/OCA/l10n-spain/blob/8.0/l10n_es_account_bank_statement_import_n43/tests/test_l10n_es_account_bank_statement_import_n43.py